ddtrace/appsec/processor.py

```diff
@@ -28,9 +28,20 @@
 log = get_logger(__name__)
 
 
-def _no_cookies(data):
+def _transform_headers(data):
     # type: (Dict[str, str]) -> Dict[str, str]
-    return {key: value for key, value in data.items() if key.lower() not in ("cookie", "set-cookie")}
+    normalized = {}
+    for header, value in data.items():
+        header = header.lower()
+        if header in ("cookie", "set-cookie"):  # TODO: Move this tuple to a frozenset ?
```

Comment:
```suggestion
        if header in ("cookie", "set-cookie"):
```

Sorry I meant that as a question for now, doesn't need to be in the code.

using a frozenset is a little more performant, but probably won't make a big difference here